mitodl/micromasters
# Check for open exchange API URL before requesting it

#### What are the relevant tickets?
Fixes #2556

#### What's this PR do?
Check for `settings.OPEN_EXCHANGE_RATES_URL` and `settings.OPEN_EXCHANGE_RATES_APP_ID` when trying to sync currency exchange rates. If either is unset, fail with a meaningful error message rather than trying to fetch an invalid URL.

#### How should this be manually tested?
Try running the app with one or both of those settings unset. Wait for the `sync_currency_exchange_rates` task to trigger, and verify that the error is meaningful.
